agent/consul/discovery_chain_endpoint.go

```diff
@@ -93,7 +93,7 @@ func (c *DiscoveryChain) Get(args *structs.DiscoveryChainRequest, reply *structs
 			reply.Index = index
 			reply.Chain = chain
 
-			if chain.IsDefault() {
```

Comment:
:thinking: Actually I think I have to pull this change out and backport it to 1.11 and 1.10